apps/cms/vite.config.ts

```
@@ -1,29 +1,95 @@
+      terserOptions: {
+        compress: {
+          drop_console: !isDev,
+          drop_debugger: !isDev,
         },
+      },
+    },
+    define: {
+      // Define process and process.env
+      'process.env': Object.entries(env).reduce((prev, [key, val]) => {
+        return {
+          ...prev,
```

Biome (1.9.4)

[error] Avoid the use of spread (`...`) syntax on accumulators.

Spread syntax should be avoided on accumulators (like those in `.reduce`) because it causes a time complexity of `O(n^2)`.
Consider methods such as .splice or .push instead.

(lint/performance/noAccumulatingSpread)